code/modules/research/designs/equipment_designs.dm

```diff
@@ -330,3 +330,13 @@
 	materials = list(MAT_SILVER = 4000, MAT_TITANIUM = 6000, MAT_URANIUM = 4000, MAT_PLASMA = 4000)
 	build_path = /obj/item/clothing/shoes/magboots/gravity
 	category = list("Equipment")
+
+/datum/design/lavarods
+	name = "Lava-Resistant Iron Rods"
```

Comment:
The name of the design is lava-resistant, but the name of the item is heat resistant?